string floorplan_name

---
string floorplan_json

bool success
string message